faces/3.0/_index.md

```diff
@@ -37,20 +37,25 @@ Please reference that ballot for the official results.
 
 ## Release Review
 
-The Specification Committee Ballot concluded successfully on 2020-mm-dd with the following results.
+The Specification Committee Ballot concluded successfully on 2020-10-21 with the following results.
 
 | Representative                                      | Representative for: | Vote |
 |-----------------------------------------------------|---------------------|------|
-| Kenji Kazumura                                      | Fujitsu             |      |
-| Dan Bandera, Kevin Sutter                           | IBM                 |      |
-| Ed Bratt, Dmitry Kornilov                           | Oracle              |      |
-| Andrew Pielage, Matt Gill                           | Payara              |      |
-| Scott Stark, Mark Little                            | Red Hat             |      |
-| David Blevins, Jean-Louis Monteiro, Cesar Hernandez | Tomitribe           |      |
-| Ivar Grimstad                                       | EE4J PMC            |      |
-| Marcelo Ancelmo, Martijn Verburg                    | Participant Members |      |
-| Werner Keil                                         | Committer Members   |      |
-| Scott (Congquan) Wang                               | Enterprise Members  |      |
-|                                                     | Total               |      |
+| Kenji Kazumura                                      | Fujitsu             |   +1   |
+| Dan Bandera, Kevin Sutter                           | IBM                 |   +1   |
+| Ed Bratt, Dmitry Kornilov                           | Oracle              |   +1   |
+| Andrew Pielage, Matt Gill                           | Payara              |   +1   |
+| Scott Stark, Mark Little                            | Red Hat             |   +1   |
+| David Blevins, Jean-Louis Monteiro, Cesar Hernandez | Tomitribe           |   +1   |
+| Ivar Grimstad                                       | EE4J PMC            |   +1   |
+| Marcelo Ancelmo, Martijn Verburg                    | Participant Members |   +1   |
+| Werner Keil                                         | Committer Members   |   +1   |
+| Scott (Congquan) Wang                               | Enterprise Members  |   +1   |
+|                                                     | Total               |   10   |
+
+Additional Votes:
+- Ed Burns: +1
+- Paul Nicolucci: +1
+
 
 The ballot was run in the [jakarta.ee-spec mailing list]()
```

Comment:
Need to provide a link to your initial post to the mailing list asking for a vote on this spec.